### DISCUSSION TITLE
移动端remotely-save+OneDrive同步错误

### DISCUSSION THREAD

**Discussion by thechosenonehaiou** [remotely-save/remotely-save#602] [Q&A]
请问移动端obsidian用remotely-save同步时，显示这个错误，是咋回事呀
![remotely-save](https://github.com/remotely-save/remotely-save/assets/48875640/2fbb6bbc-1c6c-45ad-9ebc-a58cd48b8703)

**Comment by fyears** [comment]
这里你除了 rs 还安装了什么插件呢？此外提个 bug 单我有空时候查一下吧。此外这里是 obsidian 和 rs 的版本是不是都是最新的呢？

**Comment by RanceLee233** [comment]
我也是有问题，remotely save+Onedrive。IPhone和Ipad版本的配置、插件手机上同步不过去，另外笔记部分过去了，部分没过去。另外IPhone上修改的电脑上没同步过来。

**Reply by fyears** [reply]
请问是怎么看出同步不过去的呢？是直接同步时候出错吗？是哪里同步时候出什么错呢？

**Reply by RanceLee233** [reply]
1、我之前也遇到了帖子中的400问题，重新鉴权就行了。2、同一个文件夹里，我PC端6个笔记，iPhone只有1个笔记，明显能看出问题。并且这个问题是在手动同步没有报错后发生的，但我并没有收到任何错误提示。3、解决办法我已经回在帖子最下面了，虽然不知道为什么，但的确有用

**Comment by thechosenonehaiou** [comment]
我又用过几天了，发现是能同步的，只不过在电脑端同步很快，在移动端同步很慢，可能点击同步之后得等好久，才能完成同步。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年4月15日(周一) 上午10:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [remotely-save/remotely-save] 移动端remotely-save+OneDrive同步错误 (Discussion #602)




 
我也是有问题，remotely save+Onedrive。IPhone和Ipad版本的配置、插件手机上同步不过去，另外笔记部分过去了，部分没过去。另外IPhone上修改的电脑上没同步过来。

**Reply by RanceLee233** [reply]
羡慕，为什么我过了一个周末还是老问题，并没有新的同步

**Comment by thechosenonehaiou** [comment]
因为这个要等好久才能同步呢，我现在已经基本上放弃用移动端了。




Skywalker
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "remotely-save/remotely-save"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月15日(星期一) 上午10:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [remotely-save/remotely-save] 移动端remotely-save+OneDrive同步错误 (Discussion #602)





 
羡慕，为什么我过了一个周末还是老问题，并没有新的同步

**Comment by thechosenonehaiou** [comment]
obsidian和rs插件都是最新的，我又试了几天，发现移动端是可以同步的，但是同步到移动端很慢，点击同步后，要等好久才能同步过去。下面是安装的插件





Skywalker
***@***.***



&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "remotely-save/remotely-save"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月11日(星期四) 晚上11:45
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [remotely-save/remotely-save] 移动端remotely-save+OneDrive同步错误 (Discussion #602)





 
这里你除了 rs 还安装了什么插件呢？此外提个 bug 单我有空时候查一下吧。此外这里是 obsidian 和 rs 的版本是不是都是最新的呢？

**Comment by RanceLee233** [comment]
针对我说的iPhone，ipad笔记不全的问题。发现onedrive应用-remotelysave里面的仓库数据就是不全的，解决办法是把完整的仓库拷贝进去，然后PC端直接用这个仓库，就能解决iPhone的问题了。供其他人参考。
